etc/picongpu/taurus-tud/knl_picongpu.profile.example

```diff
@@ -13,21 +13,36 @@ export MY_NAME="$(whoami) <$MY_MAIL>"
 #   - examples: "nano", "vim", "emacs -nw", "vi" or without terminal: "gedit"
 #export EDITOR="nano"
 
-# General modules #############################################################
+# Modules #############################################################
 #
-module purge
-module load cmake/3.10.1
-module load git
-module load intelmpi/2017.2.174
-
-# Compilers ###################################################################
-### ICC
-module load intel/2017.2.174
-# Boost needs to be build by yourself for now!
-# The boost root path needs to be adjusted.
-export BOOST_ROOT=$HOME/lib/boost-1.62.0
-export LD_LIBRARY_PATH=$BOOST_ROOT:$LD_LIBRARY_PATH
-export LD_LIBRARY_PATH=$BOOST_ROOT/lib:$LD_LIBRARY_PATH
+module load modenv/scs5
+module load iimpi/2018a
+module load git/2.18.0-GCCcore-6.4.0
+module load CMake/3.11.4-GCCcore-7.3.0
+module load Boost/1.66.0-intel-2018a
+module load HDF5/1.10.1-intel-2018a
+module load libpng/1.6.34-GCCcore-7.3.0
+
+# module system does not expand cmake prefix path:
```

Comment:
Imo this `expand` is not suitable here, as it does not mean expanding as for environment variable.

Reply:
export